pkgs/by-name/sp/spirv-cross/package.nix

```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation (finalAttrs: {
     python3
   ];
 
-  cmakeFlags = lib.optionals stdenv.hostPlatform.isLinux [
+  cmakeFlags = lib.optionals stdenv.hostPlatform.isUnix [
```

Comment:
This causes mass rebuild on darwin so you'll need to target the `staging` branch. I would extract this change into a separate PR, I don't have enough context about darwin to know whether it's safe.